### PR TITLE
Handle tabs in title and strings to print.

### PR DESCRIPTION
**Pull request by JalonSolov**
Tried to do as few changes as possible to existing code, while making this work.

### CODE REVIEW

File: util.go [hunk 1]
```diff
@@ -11,6 +11,11 @@ import (
 	"github.com/mattn/go-runewidth"
 )
 
+type ExpandedLines struct {
```
Comment (Delta456) [inline util.go:14]: `ExpandedLines` is public but it must be private i.e. must be renamed to `expandLines`. Also a comment will be appreciated to know what it does.

**Comment by Delta456** [comment]
Rest is fine 👍

**Comment by JalonSolov** [comment]
Done.

**Review by Delta456 — APPROVED** [review]
Great work!

**Pull request merged**